jdaviz/configs/cubeviz/plugins/viewers.py

```diff
@@ -180,6 +273,68 @@ def get_sonified_cube(self, sample_rate, buffer_size, device, assidx, ssvidx,
                                       callback=self.sonified_cube.player_callback)
         self.sonified_cube.cbuff = True
 
+        x_size = self.sonified_cube.sigcube.shape[0]
+        y_size = self.sonified_cube.sigcube.shape[1]
+
+        # Create a new entry for the sonified layer in data_lookup. The value is a dictionary
+        # containing (x_size * y_size) keys with values being arrays that represent sounds
+        self.data_lookup[results_label] = {(x, y): self.sonified_cube.sigcube[x, y, :]
+                                           for x in range(0, x_size)
+                                           for y in range(0, y_size)}
+
+        # Create a 2D array with coordinates starting at (0, 0) and going until (x_size, y_size)
+        a = np.arange(1, x_size * y_size + 1).reshape((x_size, y_size))
+
+        # Create fake WCS so the sonified layer can be added to a CCDData object and
+        # then get added to the image viewer
+        wcs = WCS({'CTYPE1': 'RA---TAN', 'CUNIT1': 'deg', 'CDELT1': -0.0002777777778,
+                   'CRPIX1': 1, 'CRVAL1': 337.5202808,
+                   'CTYPE2': 'DEC--TAN', 'CUNIT2': 'deg', 'CDELT2': 0.0002777777778,
+                   'CRPIX2': 1, 'CRVAL2': -20.833333059999998})
+
+        # Create add data with name results_label to data collection and then add it to the
+        # flux viewer
+        test = CCDData(a * u.Unit(''), wcs=wcs)
+        self.jdaviz_app.data_collection[results_label] = test
```

Comment:
```suggestion
        sonified_cube = CCDData(a * u.Unit(''), wcs=wcs)
        self.jdaviz_app.data_collection[results_label] = sonified_cube
```
I'd prefer not to leave something labeled "test" when we merge this.